### ISSUE TITLE
scriptcs is not recognized as an internal and external command

### ISSUE THREAD

**Issue by AGoh2019** [microsoft/vscode#183807]
Type: <b>Bug</b>

Do you have a workaround for this "scriptcs is not recognized as an internal and external command" while trying to run a "HelloWord.cs" program.

VS Code version: Code 1.78.2 (b3e4e68a0bc097f0ae7907b217c1119af9e03435, 2023-05-10T14:39:26.248Z)
OS version: Windows_NT x64 10.0.22621
Modes:
Sandboxed: Yes

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz (8 x 2419)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|15.77GB (8.16GB free)|
|Process Argv|--crash-reporter-id d28106b3-cd04-490a-b194-f819821f7d80|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (43)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-django|bat|1.10.0
better-toml|bun|0.3.2
markdown-julia|col|0.0.1
c-cpp-compile-run|dan|1.0.45
python-environment-manager|don|1.0.4
python-extension-pack|don|1.7.0
code-runner|for|0.12.0
c-cpp-runner|fra|5.4.0
auto-using|Fud|0.7.15
chatgpt-vscode|gen|0.0.8
fsharp-language-server|geo|0.1.28
vscode-javac|geo|0.2.45
vs-code-runner|Har|2.0.1
Ionide-fsharp|Ion|7.5.4
vscode-csharp-snippets|jor|1.1.0
language-julia|jul|1.47.2
vsc-python-indent|Kev|1.18.0
csharpextensions|kre|1.7.3
python-path|mge|0.0.14
csharp|ms-|1.25.7
python|ms-|2023.8.0
vscode-pylance|ms-|2023.5.40
cmake-tools|ms-|1.14.31
cpptools|ms-|1.15.4
cpptools-extension-pack|ms-|1.3.0
autodocstring|njp|0.6.1
java|red|1.18.0
rust-analyzer|rus|0.3.1533
rust-pack|Swe|0.3.3
tabnine-vscode|Tab|3.6.51
python-extended-snippets|tus|0.0.1
cmake|twx|0.0.17
vscode-lldb|vad|1.9.2
intellicode-api-usage-examples|Vis|0.2.7
vscodeintellicode|Vis|1.2.30
vscode-gradle|vsc|3.12.7
vscode-java-debug|vsc|0.50.0
vscode-java-dependency|vsc|0.22.0
vscode-java-pack|vsc|0.25.11
vscode-java-test|vsc|0.38.2
vscode-maven|vsc|0.41.0
vscode-spring-initializr|vsc|0.11.2
jinja|who|0.0.8

(1 theme extensions excluded)

</details>

**Comment by roblourens** [comment]
Sounds like an issue with your system configuration, make sure that tool is installed and on your path.

**Comment by AGoh2019** [comment]
I installed VS IDE first before VS Code, how should I ensure that the tool is installed properly.  Do I need to download the scriptcs is available.
--- Andrew Goh
 

    On Thursday, 1 June 2023 at 11:24:38 am SGT, Rob Lourens ***@***.***> wrote:  
 
 


Closed #183807 as completed.

**Comment by AGoh2019** [comment]
Hi Rob,
I have chocolatery installed.  How do I set set myself up for Administrative privilege access using Microsoft PowerShell?
Thanks, 
--- Andrew Goh
 

    On Thursday, 1 June 2023 at 03:04:01 pm SGT, Andrew Goh ***@***.***> wrote:  
 
 I installed VS IDE first before VS Code, how should I ensure that the tool is installed properly.  Do I need to download the scriptcs is available.
--- Andrew Goh
 

    On Thursday, 1 June 2023 at 11:24:38 am SGT, Rob Lourens ***@***.***> wrote:  
 
 


Closed #183807 as completed.
